src/styles/main.scss

```diff
@@ -1,3 +1,72 @@
+$cell-size: 100px;
+
+html {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.field {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
```

Comment:
Add a modifier for the start-day to the .calendar selector, e.g. &--start-day--#{day}.